### PR TITLE
Introduce Technology Parnter Program

### PR DESCRIPTION
**Pull request by pgoodjohn**

### CODE REVIEW

File: source/connect/overview.rst [hunk 1]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
-your account is configured as a partner account, any merchant you sign up through the OAuth onboarding flow will
-automatically be linked to your account.
-
-For more information about partnering with Mollie, please see our
-`Partner page <https://www.mollie.com/en/partners/>`_ or reach out to your Mollie partner manager.
+Technology Partner Program
+--------------------------
+
+If you are a SaaS platform, you can sign up for our Technology Partner Program to take advantage of exclusive resources,
+marketing, opportunities and dedicated partner support to help you grow your business.
```
Comment (mollierobbert) [inline source/connect/overview.rst:63]: Oxford comma: `...opportunities, and dedicated partner support...`

File: source/connect/overview.rst [hunk 2]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
-
-For more information about partnering with Mollie, please see our
-`Partner page <https://www.mollie.com/en/partners/>`_ or reach out to your Mollie partner manager.
+Technology Partner Program
+--------------------------
+
+If you are a SaaS platform, you can sign up for our Technology Partner Program to take advantage of exclusive resources,
+marketing, opportunities and dedicated partner support to help you grow your business. 
+
+Our tiered program grows alongside you, unlocking exclusive benefits as your payment volume increases, including the
```
Comment (mollierobbert) [inline source/connect/overview.rst:65]: Long sentence. Try:

`...as your payment volume increases. This includes the opportunity...`

File: source/connect/overview.rst [hunk 3]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
---------------------
-You can sign up for our referral program to receive commissions for merchants that you are onboarding to Mollie. Once
-your account is configured as a partner account, any merchant you sign up through the OAuth onboarding flow will
-automatically be linked to your account.
-
-For more information about partnering with Mollie, please see our
-`Partner page <https://www.mollie.com/en/partners/>`_ or reach out to your Mollie partner manager.
+Technology Partner Program
+--------------------------
+
```
Comment (mollierobbert) [inline source/connect/overview.rst:61]: I guess the standard across the docs codebase is not to have newlines after headings. We should probably just lint it somehow...

File: source/connect/overview.rst [hunk 4]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
+marketing, opportunities and dedicated partner support to help you grow your business. 
+
+Our tiered program grows alongside you, unlocking exclusive benefits as your payment volume increases, including the
+opportunity to earn revenue share from your customers online payments. 
+
+Once you are accepted as a partner, and your account is configured as a partner account, any merchant you sign up
+through the OAuth onboarding flow will automatically be linked to your account.
+
+For more information about partnering with Mollie and how to join the program please see our `Technology Partner Program
+<https://www.mollie.com/en/partner-programmes/saas>`_ page reach out to your partner manager.
```
Comment (mollierobbert) [inline source/connect/overview.rst:72]: Missing an **or** here.

`...page, or reach out...`

File: source/connect/overview.rst [hunk 5]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
+Our tiered program grows alongside you, unlocking exclusive benefits as your payment volume increases, including the
+opportunity to earn revenue share from your customers online payments. 
+
+Once you are accepted as a partner, and your account is configured as a partner account, any merchant you sign up
+through the OAuth onboarding flow will automatically be linked to your account.
+
+For more information about partnering with Mollie and how to join the program please see our `Technology Partner Program
+<https://www.mollie.com/en/partner-programmes/saas>`_ page reach out to your partner manager. 
+
+Partnerse note, Marketplaces are not eligible for the Technology Partner Program.
```
Comment (mollierobbert) [inline source/connect/overview.rst:74]: I think you meant 'Please note'.

Also, 'marketplaces' can be lowercase.

Comment (EszterMollie) [reply source/connect/overview.rst:74]: avoid "please" altogether

File: source/connect/overview.rst (hunk 1 above)
Comment (EszterMollie) [reply source/connect/overview.rst:63]: If you are a SaaS platform < this sounds a bit weird

File: source/connect/overview.rst [hunk 6]
```diff
@@ -56,11 +56,20 @@ fees, and receive a Mollie invoice. For this case we offer :doc:`Application fee
...
 
-Referral commissions
---------------------
-You can sign up for our referral program to receive commissions for merchants that you are onboarding to Mollie. Once
-your account is configured as a partner account, any merchant you sign up through the OAuth onboarding flow will
-automatically be linked to your account.
-
-For more information about partnering with Mollie, please see our
-`Partner page <https://www.mollie.com/en/partners/>`_ or reach out to your Mollie partner manager.
+Technology Partner Program
```
[the remark this answers is not in the document]
Comment (pgoodjohn) [reply source/connect/overview.rst:59]: I believe so, it is all capitalised also in the body of the [landing page](https://www.mollie.com/en/partner-programmes/saas)

**Pull request closed without merging**